.github/dependabot.yml

```diff
@@ -0,0 +1,516 @@
+version: 2
+registries:
+  visual-studio-securitytools:
+    type: nuget-feed
+    url: https://securitytools.pkgs.visualstudio.com/_packaging/Guardian/nuget/v3/index.json
+    # username: my-username
+    # token: ${{ secrets.NUGET_PRIVATE_REG_TOKEN }}
```

Comment:
To allow Dependabot to suggest updates for `eng/common/sdl/packages.config`:
- add username
- add token as GitHub secret with the name `NUGET_PRIVATE_REG_TOKEN`
- uncomment line 6 and 7